src/app/tests/crud/test_crud_robot_type.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.robot_type import RobotType
+from app.crud.crud_robot_type import robot_type
+from app.schemas.robot_type import RobotTypeUpdate
+from app.tests.utils.models import create_robot_type
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_crud_robot_type_create(database: AsyncSession):
+    robot_type_obj = await create_robot_type(db=database)
+    assert robot_type_obj is not None
+    assert isinstance(robot_type_obj, RobotType)
+
+
+async def test_crud_robot_type_get(database: AsyncSession):
+    robot_type_obj = await create_robot_type(db=database)
+    tmp = await robot_type.get_by_id(db=database, id=robot_type_obj.id)
+    assert robot_type_obj.__eq__(tmp)
+
+
+async def test_crud_robot_type_get_multi(database: AsyncSession):
+    await create_robot_type(db=database)
+    await create_robot_type(db=database)
+    robot_types = await robot_type.get_multi(db=database)
+    assert len(robot_types) == 2
+
+
+async def test_crud_robot_type_update(database: AsyncSession):
+    robot_type_obj = await create_robot_type(db=database)
+    await robot_type.update(db=database, db_obj=robot_type_obj, obj_in={"name": "modified"})
+    assert robot_type_obj.name == "modified"
+
+    await robot_type.update(db=database, db_obj=robot_type_obj, obj_in=RobotTypeUpdate(name="modified again"))
+    assert robot_type_obj.name == "modified again"
```

Comment:
Please also assert that the id has not changed.

Reply:
See other comment